OmniKit/MessageTransport/MessageBlocks/PodInfoFaultEvent.swift

```diff
@@ -38,7 +24,7 @@ public struct PodInfoFaultEvent : PodInfo, Equatable {
     public let secondaryLoggedFaultEvent: FaultEventCode
     public let logEventError: Bool
     public let reservoirStatusAtFirstLoggedFaultEvent: ReservoirStatus
-    public let recieverLowGain: UInt8
+    public let receiverLowGain: UInt8
     public let radioRSSI: UInt8
```

Comment:
This can be turned into a signed integer, so we can represent the RSSI as negative numbers.